src/gamma_inc.jl

```diff
@@ -142,7 +141,250 @@ function rgammax(a::Float64,x::Float64)
             return (a*exp(t))*(1.0 + rgamma1pm1(a))
         end
     end
+end
+"""
+    auxgam(x)
+
+Compute function `g` in ``1/\\Gamma(x+1) = 1+x*(x-1)*g(x)``, -1 <= x <= 1 
+"""
+function auxgam(x::Float64)
+    dr = zeros(18)
+    if x < 0
+        return -(1.0+(1+x)*(1+x)*auxgam(1+x))/(1.0-x)
+    else
+        dr[1] = -1.013609258009865776949
```

Comment:
like my comment below, this could be moved to a `const` array outside the function.